avocado/core/nrunner.py

```diff
@@ -653,12 +674,15 @@ def __init__(self, identifier, runnable, status_uris=None,
         if known_runners is None:
             known_runners = {}
         self.known_runners = known_runners
+        self.prereqs = []
         self.spawn_handle = None
         self.output_dir = None
 
     def __repr__(self):
-        fmt = '<Task identifier="{}" runnable="{}" status_services="{}"'
-        return fmt.format(self.identifier, self.runnable, self.status_services)
+        fmt = ('<Task identifier="{}" runnable="{}" prereqs="{}"'
+               ' status_services="{}"')
+        return fmt.format(self.identifier, self.runnable, self.prereqs,
+                          self.status_services)
```

Comment:
Could this representation be an f-string? It seems more readable if we have dropped support for Python 3.5.

Reply:
We've been using `format` all over the code; that is the only reason I'm still using it when I have to.

Reply:
I see, it is never too late to start a possible conversion but if somebody is actually planning for a one-time run over all code one day consistency here is fine.

Reply:
Indeed, I think a refactoring PR on f-string would be better.

Reply:
Might also be worth noting it down as a TODO for the future but it's your call, just advising on keeping track of ideas in one place for everybody.

Reply:
Yep, will do it after this work is merged.